packages/synapse-react-client/src/lib/style/components/_all.scss

```diff
@@ -10,16 +10,16 @@
   './expandable_content', './bar-loader', './facet-plots-card',
   './featured-data-plots', './featured-data-tabs', './programs',
   './user-card-list-groups', './evaluation_queue/all',
-  './personal-access-tokens', './copy-to-clipboard', './account-level-badge',
```

Comment:
Remove `./copy-to-clipboard` - no longer used, instead style `CopyToClipboardInput` directly